### PR TITLE
Documented the local search box for dumped contents

### LINKED ISSUE #7381: Document the search feature of the HTML dumper
see symfony/symfony#21109 and symfony/symfony#21238

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #7381.

### CODE REVIEW

**Comment by ogizanagi** [comment]
New keyboard shortcuts were added in https://github.com/symfony/symfony/pull/21238 in order to navigate between found occurrences, replicating browsers behavior.
Should we document that too? 😃

File: components/var_dumper.rst [hunk 1]
```diff
@@ -105,6 +105,16 @@ This behavior can be changed by configuring the ``dump.dump_destination``
...
 :doc:`the DebugBundle configuration reference </reference/configuration/debug>`.
 
+.. tip::
+
+    .. versionadded:: 3.3
+        The local search box was introduced in Symfony 3.3.
+
+    If the dumped contents are complex, consider using the local search box to
+    look for specific variables or values. First, click anywhere on the dumped
+    contents and then press ``Ctrl. + F`` or ``Cmd. + F`` to make the local
```
Comment (ogizanagi) [inline components/var_dumper.rst]: I think Sphinx is supposed to support keystrokes sequences: http://www.sphinx-doc.org/en/stable/markup/inline.html#role-kbd

What would it look like with this?

**Comment by ogizanagi** [comment]
👍 Thanks :)

Status: Reviewed

**Comment by xabbuh** [comment]
Thank you Javier.

**Pull request closed without merging**